blueocean-web/package.json

```diff
@@ -8,7 +8,8 @@
     "test": "gulp test",
     "test:watch": "gulp test:watch",
     "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch"
+    "bundle:watch": "gulp bundle:watch",
+    "postinstall": "node ../bin/cleanshrink.js"
```

Comment:
Yeah, cleaning the shrink unfortunately is something we need to do to fight against the default behavior of shrinkwrap. I changed the Jenkinsfile so that it validates the shrink before even running the main build, so if a developer commits an untidy shrink it will fail almost immediately.

Unfortunately the hooks we have in npm are limited: postinstall doesn't run if you install a specific dependency, only when doing `npm install` with no args. If anyone else has ideas I'm all ears.